custom/icds_reports/utils/__init__.py

```diff
@@ -99,7 +100,8 @@ def _get_static_report_configuration_without_owner_transform(self, report_id, do
                 report_column.transform = {}
         return static_report_configuration
 
-    def data(self):
+    @quickcache(['cache_key'], timeout=60 * 60)
```

Comment:
Is this intended to be cached for one report load? I think 60 minutes is quite a long time for that. Ideally just a minute or two should be ok, but 5 more than sufficient right?

Reply:
Yeah, mostly for one report load. I put it high since the district level reports were taking upto an hour (could be useful while doing this as email report). But, not sure it will be of use, since ideally we don't want these reports to an hour to take. We can make it 5.